deploy/quickstart/quick_start.py

```diff
@@ -0,0 +1,245 @@
+import os
+from os import path
+import urllib.request
+from subprocess import call
+import subprocess
+
+
+def quick_start():
+    version = 'v0.2.0'
+
+    home = os.getenv('HOME')
+    install_dir = request_input_path('Select an installation directory',
+                                     '{0}/jmui'.format(home))
+
+    bin_dir = '{0}/bin'.format(install_dir)
+    make_or_replace_or_reuse_directory(bin_dir, exit_if_invalid=True)
+
+    config_dir = '{0}/config'.format(install_dir)
+    make_or_replace_or_reuse_directory(config_dir, exit_if_invalid=True)
+
+    os.chdir(install_dir)
+
+    service_type = request_input_from_list('Select a shim to use',
+                                           ['Cromwell', 'dsub'], 'Cromwell')
+    options = {'Cromwell': quick_start_cromwell, 'dsub': quick_start_dsub}
+    options[service_type](version, install_dir, bin_dir, config_dir)
+
+
+def quick_start_cromwell(version, install_dir, bin_dir, config_dir):
+    os.chdir(bin_dir)
+
+    capabilities_config = download_file(
+        version, 'deploy/cromwell/capabilities-config.json',
+        'capabilities-config.json')
+    docker_compose = download_file(
+        version,
+        'deploy/cromwell/docker-compose/cromwell-compose-template.yml',
+        'docker-compose.yml')
+    nginx = download_file(version, 'deploy/cromwell/docker-compose/nginx.conf',
+                          'nginx.json')
+    ui_config = download_file(version, 'deploy/ui-config.json',
+                              'ui-config.json')
+
+    os.chdir(config_dir)
+    api_config = download_file(version, 'deploy/cromwell/api-config.json',
+                               'api-config.json')
+    cromwell_url = request_input_with_help(
+        'Enter the raw Cromwell IP address, eg "10.1.10.100" (DO NOT USE "LOCALHOST" or "127.0.0.1"!)',
+        cromwell_service_clue)
+
+    os.chdir(install_dir)
+    start_script_file = '{0}/jmui_start.sh'.format(bin_dir)
+    start_script_contents = [
+        'if docker ps | grep job-manager', 'then',
+        'echo \'Stopping current instance(s)\'',
+        'docker stop $(docker ps | grep \'job-manager\' | awk \'{ print $1 }\')',
+        'fi', 'export CROMWELL_URL=http://{0}:8000/api/workflows/v1'.format(
```

Comment:
I'd let users specify the port number(maybe as part of the IP address) here as well (we can still leave 8000 as a default value)